lib/cdo/honeybadger.rb

```diff
@@ -3,6 +3,7 @@
 
 # Honeybadger extensions for command error logging
 module Honeybadger
+  @@code_org_team_id = 530
```

Comment:
should this be a constant? Either way, I'd love to see a comment explaining what this represents and how this value was determined (and how to re-determine it in the future, if necessary)